flink-kubernetes-operator-api/src/main/java/org/apache/flink/kubernetes/operator/crd/status/ReconciliationMetadata.java

```diff
@@ -15,10 +15,9 @@
  * limitations under the License.
  */
 
-package org.apache.flink.kubernetes.operator.reconciler;
+package org.apache.flink.kubernetes.operator.crd.status;
```

Comment:
I suggest keeping this out of the `crd` package. It is internal to the operator logic and moving it under the CRD pollutes the CRD ref docs

Reply:
Good catch, done.